src/Skeletor/App/Config/skeletor.yml

```diff
@@ -5,9 +5,11 @@ frameworks:
     - Laravel54Framework
     - LaravelLumen54Framework
 packages:
```

Comment:
I had thought about it but both Laravel and Lumen add PHPUnit as a dev dependency so don't think we have to add anything else for now. I'm thinking of another PR where you can separate normal and dev dependencies, maybe we can add some more testing packages then.